recipes/rgi/meta.yaml

```diff
@@ -27,13 +27,20 @@ requirements:
   run:
     # RGI requires these exact versions of blast prodigal and diamond, or else --alignment_mode DIAMOND fails
     - python
```

Comment:
So, when I run the tests, I get the following messages:

```
+ rgi load -h
Traceback (most recent call last):
  File "/opt/conda/conda-bld/rgi_1538772728787/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/bin/rgi", line 2, in <module>
    from app.MainBase import MainBase
  File "/opt/conda/conda-bld/rgi_1538772728787/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python2.7/site-packages/app/MainBase.py", line 9, in <module>
    import app.build_kmer_sets
  File "/opt/conda/conda-bld/rgi_1538772728787/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python2.7/site-packages/app/build_kmer_sets.py", line 2, in <module>
    import app.make_kmer_json
  File "/opt/conda/conda-bld/rgi_1538772728787/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python2.7/site-packages/app/make_kmer_json.py", line 145
    f = {**f, **results[i][1]}
          ^
SyntaxError: invalid syntax
```

A syntax error makes me think an issue with the version of Python. I also noticed that no python version is specified here, and it looks like Python 2 is being installed:

```
    python:                        2.7.15-h33da82c_1     conda-forge
```

Could you try specifying to use python >= 3.6 here and in the **host** section (which the rgi docs say is required https://github.com/arpcard/rgi/tree/master#requirements).

This may make the other error you encountered go away.